conf/prod.routes

```diff
@@ -26,3 +26,5 @@ GET           /customs-declarations-stub/admin/notification/:clientId/:operation
 POST          /customs-declarations-stub/admin/notification/:clientId/:operation/:lrn        uk.gov.hmrc.customs.declarations.stub.controllers.NotificationController.addNotification(clientId, operation, lrn)
 
 DELETE        /customs-declarations-stub/admin/notification/:id                              uk.gov.hmrc.customs.declarations.stub.controllers.NotificationController.deleteNotification(id)
+
+GET           /eori/:eori/verified-email                                                     uk.gov.hmrc.customs.declarations.stub.controllers.CustomsDataStoreStubController.emailIfVerified(eori)
```

Comment:
Although it is not super important, it might be worth making this URL as close to the real one as possible. Do we know what is the real URL going to be?

Reply:
It's already the real one.
https://jira.tools.tax.service.gov.uk/browse/MMCA-2122.